source/exe/main_common.cc

```diff
@@ -75,6 +76,11 @@ MainCommonBase::MainCommonBase(OptionsImpl& options, Event::TimeSystem& time_sys
     server_ = std::make_unique<Server::InstanceImpl>(
         options_, time_system, local_address, test_hooks, *restarter_, *stats_store_,
         access_log_lock, component_factory, std::move(random_generator), *tls_);
+
+    if (options.mutexTracingEnabled()) {
+      Envoy::MutexTracer::getOrCreateTracer();
```

Comment:
I had though this was done inside the InstanceImpl ctor.  Can we move it there and also save the MutexTracer* for accessing in admin later?

Reply:
Done in 677c64a.